src/unix/linux_like/linux/mod.rs

```diff
@@ -2355,7 +2355,7 @@ pub const MAP_FIXED_NOREPLACE: ::c_int = 0x100000;
 // uapi/linux/vm_sockets.h
 pub const VMADDR_CID_ANY: ::c_uint = 0xFFFFFFFF;
 pub const VMADDR_CID_HYPERVISOR: ::c_uint = 0;
-pub const VMADDR_CID_RESERVED: ::c_uint = 1;
+pub const VMADDR_CID_LOCAL: ::c_uint = 1;
```

Comment:
We should add `#[deprecated]` attribute instead of removing for backward compatibility (even if it's unused).

Reply:
And CI images don't have `VMADDR_CID_LOCAL` as they're old a bit so we should also tweak the test to skip this.

Reply:
Sure! I'll do it!
Thanks!

Reply:
> And CI images don't have `VMADDR_CID_LOCAL` as they're old a bit so we should also tweak the test to skip this.

Where can I find the tests?

Reply:
Here's the test file: https://github.com/rust-lang/libc/blob/master/libc-test/build.rs
And you should tweak here:
https://github.com/rust-lang/libc/blob/f3bc8363593e6e62a5229c7fe22192961330e0e7/libc-test/build.rs#L2558

Reply:
Should I skip also VMADDR_CID_RESERVED?

E.g. on my host the test fails because it is not defined anymore.

Reply:
Yes good catch, we should certainly (actually the test fails on my local as well :P).